testscommon/goroutines/filter.go

```diff
@@ -9,6 +9,8 @@ var ignorable = []string{
 	"github.com/beevik/ntp.getTime",
 	"/usr/local/go/src/net/dnsclient_unix.go",
 	"/usr/local/go/src/runtime/proc.go",
+	"net._C2func_getaddrinfo",
```

Comment:
sometimes the tests that count go routines started "capture" some go routines that do other, unrelated stuff regarding the tests, such as DNS resolving